environment-dev.yml

```diff
@@ -1,17 +1,18 @@
 name: sparrow
 channels:
-  - conda-forge
+    - conda-forge
 dependencies:
-  # Build
-  - cmake
-  - ninja
-  # Tests
-  - doctest
-  # P0355R7 (Extending chrono to Calendars and Time Zones) has not been entirely implemented in libc++ yet.
-  # See: https://libcxx.llvm.org/Status/Cxx20.html#note-p0355
-  # For now, we use HowardHinnant/date as a replacement if we are compiling with libc++.
-  # TODO: remove this once libc++ has full support for P0355R7.
-  - howardhinnant_date
-  # Documentation
-  - doxygen
-  - graphviz
+    # Build
+    - cmake
+    - ninja
+    # Tests
+    - doctest
+    - catch2
+    # P0355R7 (Extending chrono to Calendars and Time Zones) has not been entirely implemented in libc++ yet.
+    # See: https://libcxx.llvm.org/Status/Cxx20.html#note-p0355
+    # For now, we use HowardHinnant/date as a replacement if we are compiling with libc++.
+    # TODO: remove this once libc++ has full support for P0355R7.
+    - howardhinnant_date
+    # Documentation
+    - doxygen
+    - graphviz
```

Comment:
I'm not sure what is the issue but can you revert the indent be sure that you have blank line ?
https://github.com/man-group/sparrow/actions/runs/12066345620/job/33646992095?pr=301